metaworld/benchmarks/ml1.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, task_name, env_type='train', n_goals=50, sample_all=False):
         else:
             raise NotImplementedError
 
-        args_kwargs[task_name]['random_init'] = False
+        args_kwargs[task_name]['kwargs']['random_init'] = False
```

Comment:
This should only change a copy of the dictionary, or other benchmarks that use this dictionary will change behavior.

Reply:
Actually, nevermind. I can't read, apparently.